src/admin/components/elements/ReactSelect/MultiValue/index.tsx

```diff
@@ -51,7 +51,6 @@ export const MultiValue: React.FC<MultiValueProps<OptionType>> = (props) => {
         onMouseDown: (e) => {
           if (!disableMouseDown) {
             // we need to prevent the dropdown from opening when clicking on the drag handle, but not when a modal is open (i.e. the 'Relationship' field component)
-            e.preventDefault();
```

Comment:
I think we should remove this change, https://github.com/payloadcms/payload/pull/2870 likely is the permanent fix for this.